crates/turbopack-css/src/process.rs

```diff
@@ -131,7 +134,10 @@ impl<'i, 'o> StyleSheetLike<'i, 'o> {
                 })?;
 
                 if let Some(srcmap) = &mut srcmap {
+                    debug_assert_eq!(ss.sources.len(), 1);
```

Comment:
Would this break for an empty css file?

Reply:
No, as `sources` is the list of file names.